directembedding/src/main/scala/ch/epfl/directembedding/transformers/DSLVirtualization.scala

```diff
@@ -0,0 +1,23 @@
+package ch.epfl.directembedding.transformers
+
+import ch.epfl.directembedding.{ DirectEmbeddingUtils, DirectEmbeddingModule }
+import ch.epfl.yinyang.EmbeddedControls
+
+object RewireEmbeddedControls extends EmbeddedControls
+
+trait DSLVirtualization extends DirectEmbeddingModule with DirectEmbeddingUtils {
+  import c.universe._
+
+  def DSLVirtualizer = new (Tree => Tree) {
+    def apply(tree: Tree) = {
+      c.typecheck(q"""
+      import _root_.ch.epfl.directembedding.transformers.RewireEmbeddedControls._
+      {
+          import ${c.parse(virtualizationConfig)}._
```

Comment:
Neat :)  However, I think it would be better to just give a type of the object and extract the path from it. That way it is typesafe, you can give a nice error message, etc.

Failing use-case, users that make a typo in the config will not be able to detect the error easily.